cmd/snap/cmd_advise.go

```diff
@@ -205,11 +210,50 @@ func adviseViaAptHook() error {
 	return nil
 }
 
+type Snap struct {
```

Comment:
`advisor.Package` would've served you as well

Reply:
Indeed that's the same as _advisor.Command_ I think.. unless there's a good reason to have something else, let's please use it.